src/main/java/ro/vladutit/Don/t/forget/v2/controller/DashboardController.java

```diff
@@ -18,7 +19,8 @@
     public CategoryService categoryService;
 
     @RequestMapping({"/", "/index"})
-    public String index() {
+    public String index(Model user) {
```

Comment:
this should be called 'register' instead of 'index', right?

Reply:
Not at all. Register is just a box, not the whole page. The whole page and most of it is the first page, ie index. This has been the case so far, only now I have added another line that creates a new user for me. For this change, the method does not have to be called a register.